src/lib.js

```diff
@@ -59,25 +58,7 @@ import EnhancedTableHead from "./util/table/EnhancedTableHead";
 import TablePaginationActions from "./util/table/TablePaginationActions";
 import { stableSort, getSorting } from "./util/table/TableSort";
 
-import { createMuiTheme, MuiThemeProvider } from "@material-ui/core/styles";
-
-const getMuiTheme = (theme) =>
-    createMuiTheme({
-        palette: {
-            primary: {
-                main: palette.blue,
-            },
-            secondary: {
-                main: palette.lightBlue,
-            },
-        },
-        typography: {
-            button: {
-                textTransform: "none",
-            },
-            useNextVariants: true,
-        },
-    });
+import getMuiTheme from "./util/CyVerseTheme";
```

Comment:
I forget how the convo went now, but don't we want to export this so sonora can access it?

Reply:
It's exported farther down in the file, looks like.

Reply:
Oops, you're right - I had to view the whole file to see it since it wasn't a code change.